Automattic/sensei
# Add Exit Survey

Fixes #2374

### Changes proposed in this Pull Request

* When clicking `Deactivate` on the plugins screen, open up a modal with an exit survey form
* Add AJAX action to save the submitted feedback and send track event 
* Send the event anonymously, with a hashed site ID, when usage tracking is disabled

### Testing instructions

* Go to the Plugins screen, and click `Deactivate` on the Sensei LMS plugin
* A modal should open up
* Clicking `Skip Feedback` should continue with deactivating plugin
-- 
* Do it again, select a reason, type in details and submit
* A `sensei_plugin_deactivate` track event should be sent, with the selected reason and details


### Screenshot / Video

<img width="590" alt="image" src="https://user-images.githubusercontent.com/176949/96722681-ef4cf480-13ad-11eb-91bf-3ee872ce26fc.png">


![QuickTime movie](https://user-images.githubusercontent.com/176949/96722578-d2182600-13ad-11eb-83bf-139df8c5f35d.gif)
